source/clear-linux/get-started/bare-metal-install-server/bare-metal-install-server.rst

```diff
@@ -0,0 +1,857 @@
+.. _bare-metal-install-server:
+
+Install |CL-ATTR| on bare metal with live server
+################################################
+
+These instructions guide you through installing |CL-ATTR| on bare metal from a bootable USB drive using a live server image.
+
+.. contents::
+   :local:
+   :depth: 1
+
+System requirements
+*******************
+
+Assure that your target system supports the installation:
+
+* :ref:`system-requirements`
+* :ref:`compatibility-check`
+
+Download the latest |CL| live server image
+******************************************
+
+Get the latest |CL| installer image from the `downloads page`_. Look for the
+:file:`clear-[version number]-live-server.iso` file.
+
+#. Follow your OS instructions to create a bootable USB drive.
+
+   * :ref:`bootable-usb`
+
+#. After downloading the image, verify and decompress the file per your OS.
+
+Install |CL| on your target system
+**********************************
+
+We formatted the previously created USB drive as a UEFI boot device. Our
```

Comment:
Technically, the USB contains an ISO image that will boot with either UEFI or legacy BIOS, but you must enable UEFI to install from the live image.
I would drop the line "We formatted the previously created USB drive as a UEFI boot device."
You could say something like "Ensure your system is configured to boot UEFI."

Reply:
Done. I revised that entire paragraph. It was a holdover from before my time. See latest commit.